readme.md

```diff
@@ -8,16 +8,16 @@ to be executed in their programs.
 Currently must be installed from git
 
 ```
-npm i --save https://github.com/zazuko/rdf-native-loader-code
-npm i --save https://github.com/zazuko/rdf-native-loader
+npm i --save https://github.com/zazuko/rdf-loader-code
+npm i --save https://github.com/zazuko/rdf-loaders-registry
```

Comment:
`rdf-loaders-registry` is not a dependency? Maybe peer dependency? @vhf

Reply:
If rdf-loader-code cannot be used without rdf-loaders-registry but doesn't `require` it, peer dependency is the right choice IMO :)

Reply:
It's possible to use a loader without the registry and it makes sense for known structures to enforce a specific result. So I don't think a peer dependency should be added.